src/azul/pagination.py

```diff
@@ -0,0 +1,35 @@
+from azul.requests import parse_url_qs
+
+internal_pagination_parameters = ('search_after', 'search_after_uid', 'search_before', 'search_before_uid')
+'''
+    These pagination parameters are used internally, but should removed from the response that a client receives.
```

Comment:
If you looked around, you'd see that we don't indent docstrings like that. And docstrings are always triple double quoted.

Reply:
Wrap documentation on 80.